configure.ac

```diff
@@ -1471,7 +1472,7 @@ fi
 
 dnl libevent check
 
-if test "$build_bitcoin_cli$build_bitcoind$bitcoin_enable_qt$use_tests$use_bench" != "nonononono"; then
+if test "$build_bitcoin_cli$build_bitcoind$bitcoin_enable_qt$enable_fuzz_binary$use_tests$use_bench" != "nonnoononono"; then
```

Comment:
i sometimes wonder if we can switch to actual logic instead of this hilarious ever growing nononononono concatenation
(outside scope of this PR obviously)

Reply:
If we used true/false instead of yes/no, we could just do

```sh
if $build_bitcoin_cli || $build_bitcoind || $bitcoin_enable_qt || $enable_fuzz_binary || $use_tests || $use_bench; then
```

Or even if we just add a:

```sh
yes() { true; } no() { false; }
```
somewhere...